### PR TITLE
Enabled path-keys-no-trailing-slash

### PR DESCRIPTION
**Pull request by philsturgeon**
Updated our resolver with some changes to oas-linter (as hopefully one day they will merge). 

This specific change allows us to enable a rule that's been disabled for a while.

### CODE REVIEW

File: rules/default.json [hunk 1]
```diff
@@ -38,9 +38,9 @@
         },
         {
             "name": "path-keys-no-trailing-slash",
-            "object": "pathInfo<--This doesnt exist so this whole rule is broke",
-            "enabled": false,
-            "description": "path item keys should not end with a slash",
+            "object": "openapi",
```
Comment (philsturgeon) [inline rules/default.json]: I took this from [oas-linter/rules.yaml](https://github.com/Mermade/oas-kit/blob/master/packages/oas-linter/rules.yaml) but im not sure its right. These aren't keys on `openapi` they're multiple keys on `paths`. Going to have to defer to @MikeRalphson on this one.

Comment (MikeRalphson) [reply rules/default.json]: Yes, this was broken in multiple ways. :disappointed: The fixes are here: https://github.com/Mermade/oas-kit/commit/f17105c66e2dbd48e9c0e0d46c263cf65d7ce506

Comment (MikeRalphson) [reply rules/default.json]: You'll need to bump your dep on `oas-validator` to `^1.1.7`

**Comment by philsturgeon** [comment]
That fixed it all up, thanks @MikeRalphson! 

```
#/paths/~1pet~1  R: path-keys-no-trailing-slash  D: paths should not end with a slash
expected '/pet/' not to end with '/' (false negative fail)

More information: https://speccy.io/rules/#path-keys-no-trailing-slash
```

**Pull request merged**